services/agents/registry.go

```diff
@@ -488,6 +488,24 @@ func (r *Registry) CheckConnectionToService(ctx context.Context, service *models
 	resp := pmmAgent.channel.SendRequest(request)
 	l.Infof("CheckConnection response: %+v.", resp)
 
+	if service.ServiceType == models.MySQLServiceType {
```

Comment:
I'm not sure that it should be here.

Reply:
Added https://jira.percona.com/browse/PMM-4932 to track that

Reply:
cool, thanks